### PR TITLE
Release anouncement for filter

### PR DESCRIPTION
**Pull request by wallner**
This pull request adds the filtering of resources to the release announcement. It also put the complete document into the passive voice.

### CODE REVIEW

**Comment by tkrille** [comment]
Regarding [92d2f10](https://github.com/osiam/osiam.github.io/pull/12/commits/92d2f10fa21b1d754d6a8d7b12fc0e65962c4202): The commit message does not really reflect the changes. Do you really mean "passive"? Because I don't see only 1 change to a passive form. It's more like "Various improvements to the release announcement". Btw, I like the improvements and we should "backport" them to the other announcements.

**Comment by wallner** [comment]
Ok, I will reword the commit message.

**Comment by tkrille** [comment]
Can you add a simple example to better explain the purpose of attribute filtering?

**Comment by wallner** [comment]
Ok, I will give it a try.

File: _drafts/release-connector4java-1.9.md [hunk 1]
```diff
@@ -18,33 +18,38 @@ You can also download the JAR files using one of the following links:
...
-parameter list.
+OSIAM 3.0 makes it possible to filter the attributes of the returned resources.
+To enable this feature in the connector the methods `getUser()`, `getAllUsers()`, `getMe()`, `getGroup()` and `getAllGroups()` now take the attributes requested as optional final String varargs in their parameter list.
 
+Retrieving a list of all `User` resources with only their userName and their
+creation date can be realized as follows:
+
+```java
+List<User> users = osiamConnector.getAllUsers(accessToken, "userName",
+"meta.created");
```
Comment (tkrille) [inline _drafts/release-connector4java-1.9.md]: No line break, please.

Comment (tkrille) [inline _drafts/release-connector4java-1.9.md]: Or at least one, that conforms so the Java coding conventions :)

**Comment by tkrille** [comment]
You put the example into the wrong commit.

**Comment by wallner** [comment]
Yes. That is unfortunate, but I don't see how it is relevant in a release announcement. I'm only going to fix it if you absolutely insist.

**Comment by tkrille** [comment]
Yes, please do so. At least for the sake of practicing your Git skills :)

**Comment by wallner** [comment]
I don't really need to practice my Gti skills and I really don't  see the point, other than to satisfy your ocd, but here you go. Happy?

**Comment by tkrille** [comment]
I'm very happy. Thank you very much!

**Pull request merged**